tests/test_wordpress_api.py

```diff
@@ -221,7 +228,28 @@ def test_getting_articles_from_last_year(self, get):
             + "&tags_exclude="
             + "&categories="
             + "&exclude="
+            + "&author="
             + "&before=2007-12-05"
             + "&after=2006-12-05"
         )
         self.assertEqual(article, (["hello_test"], 12))
+
+    @patch("canonicalwebteam.http.CachedSession.get")
+    def test_getting_articles_from_author(self, get):
```

Comment:
you should test fails too.

Reply:
I have added a very basic failing test. 

The testing needs an overhaul though. Id rather redo it porperly than spent too much time here in this PR.